package.json

```diff
@@ -10,6 +10,7 @@
     "serve": "./entrypoint 0.0.0.0:${PORT}",
     "test": "npm run python-lint && npm run test-python && sass-lint static/**/*.scss --verbose --no-exit",
     "test-python": "python3 -m unittest discover tests",
+    "test-links": "./entrypoint 0.0.0.0:${PORT} && sleep 2 && linkchecker --threads 2 --ignore-url /store --no-warnings http://0.0.0.0:${PORT}",
```

Comment:
this actually can be removed or thought in another way since the test-links is blocked on the first command `./entrypoint`. I think just keeping it on the CI might be enough